docs/sources/write/front-matter/index.md

```diff
@@ -283,23 +282,73 @@ Use `weight` to specify a different topic order within the left-hand sidebar on
 
 Use increments of `100` for content files. Doing so makes it easier for you to re-order existing topics when you add new topics. Weights are per directory.
 
-## Example with different page and menu titles
+## Tutorials
 
+There is additional front matter that you only need for tutorials.
+Tutorials should also include all the regular front matter.
+
+### Associated technologies
+
+The `associated_technologies` front matter is a sequence of strings that refer to taxonomies in the website data directory.
+If you are a Grafana Labs employee, you can add view the associated technologies in the [website data directory](https://github.com/grafana/website/tree/master/data/taxonomies/associated_technologies).
```

Comment:
"add view" looks wrong. Should this be "add or view" or "add View" or "add `view`"? Same comment applies below.

Reply:
```suggestion
If you are a Grafana Labs employee, you can view the associated technologies in the [website data directory](https://github.com/grafana/website/tree/master/data/taxonomies/associated_technologies).
```